addons/website_crm_partner_assign/controllers/main.py

```diff
@@ -34,10 +34,19 @@ def get_domain_my_opp(self, user):
 
     def _prepare_home_portal_values(self, counters):
         values = super()._prepare_home_portal_values(counters)
+        crm_lead = request.env['crm.lead']
```

Comment:
We usually do Upper-camelcase for empty recordsets that are only used as class instances: `CrmLead`